src/app/search/search-tool-table/search-tool-table.component.html

```diff
@@ -32,9 +32,16 @@
         <span *ngIf="tool.private_access">
           <app-private-icon></app-private-icon>
         </span>
-        <a [matTooltip]="tool?.tool_path" [routerLink]="'/containers/' + tool.tool_path">{{
-          tool?.namespace + '/' + tool?.name + (tool?.toolname ? '/' + tool?.toolname : '')
-        }}</a>
+        <div *ngIf="isAppTool(tool)">
```

Comment:
Methods risk being invoked a lot... I don't know if a pipe, if you can use one here, would be better. Or a component? Gary will know.

Reply:
pipe should be better than this